src/async_impl/client.rs

```diff
@@ -2071,6 +2083,23 @@ impl ClientBuilder {
         self
     }
 
+    /// Override the default congestion control algorithm to use [BBR]
+    ///
+    /// The default congestion control algorithm in [`quinn`] is [CUBIC]. This method overrides the
```

Comment:
```suggestion
    /// The current default congestion control algorithm is [CUBIC]. This method overrides the
```

Also, I'd remove the reference about it being experimental in quinn (all of http3 is experimental in reqwest XD).

Reply:
Done. Please take another look